### DISCUSSION TITLE
[Résolu] Fonctionnement et optimisation des svg

### DISCUSSION THREAD

**Discussion by thomasCCCC** [cnumr/EcoIndex#275] [Q&A]
Bonjour à tous, 
J'obitiens le message 'Optimiser les images svg'
L'image https://kolcom.fr/wp-content/uploads/2023/10/kolcom-kastorgreen-1-opti.svg est pourtant passée dans la moulinette svgo et l'erreur s'affiche encore. 
Quelqu'un pourrait m'expliquer comment sont analysés les svg pour améliorer mes images et faire disparaitre cette erreur ? :)
Merci d'avance.
Thomas

**Comment by vvatelot** [comment]
Bonjour @thomasCCCC je ne suis pas sur de tout suivre. De quel outil parles tu ?

**Reply by thomasCCCC** [reply]
Bonjour Vincent, merci pour ta réponse il s'agit de l'extension Google greenit analysis et de l'outil svgo pour l'optimisation des fichiers svg. Après avoir vidé le cache litespeed et le cache du navigateur tout fonctionne parfaitement. Merci encore

**Reply by vvatelot** [reply]
Bonjour @thomasCCCC merci pour le complément d'information. Effectivement, avec Greenit Analysis, il faut bien vider le cache du navigateur. Plus d'infos dans la doc: https://github.com/cnumr/GreenIT-Analysis#utiliser-lextension
Du coup c'est OK pour toi ?

**Comment by thomasCCCC** [comment]
Bonsoir Vincent,
Oui c'est tout bon ! Merci pour ce merveilleux outils !!

Le jeu. 12 oct. 2023 à 09:22, Vincent Vatelot ***@***.***> a
écrit :

**Reply by vvatelot** [reply]
Cool, tant mieux ! Mais tout le mérite revient à @didierfred !

**Reply by tbroyer** [reply]
Rendons à César… : les différentes règles viennent initialement d'EcoMeter.
